response.go

```diff
@@ -16,11 +16,14 @@ func (ctx *RequestCtx) JSONResponse(body interface{}, statusCode ...int) (err er
 	}
 
 	var data []byte
-
+	jsonMarshaller := ctx.jsonMarshaller
```

Comment:
I think this is a dead code, because the server handler always set the json marshaler configured in the server.
But you must set the default marshaler into AcquireRequestCtx() function